commune.lic

```diff
@@ -0,0 +1,367 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#commune
+=end
+
+custom_require.call(%w(common common-items common-arcana drinfomon))
+
+class Commune
+  include DRC
+  include DRCI
+  include DRCA
+
+  ######################## ITEM ACTIONS ########################
+
+  def get_cleric_item(item)
```

Comment:
Does `get_cleric_item` need to return true or false? I don't see anywhere the return value is used.

Reply:
It does not need to return true or false. I changed it so if the item is missing it will exit the script.

Reply:
I also changed stow_cleric_item to no longer return true or false. Same reasoning.